tests/Oryx.BuildImage.Tests/VersionInformationTest.cs

```diff
@@ -103,6 +103,7 @@ public void DotNetAlias_UsesLtsVersion_ByDefault(string buildImageName)
         [InlineData("2.2", DotNetCoreSdkVersions.DotNetCore22SdkVersion)]
         [InlineData("3", DotNetCoreSdkVersions.DotNetCore30SdkVersion)]
         [InlineData("3.0", DotNetCoreSdkVersions.DotNetCore30SdkVersion)]
+        [InlineData("3.1", DotNetCoreSdkVersions.DotNetCore31SdkVersionPreviewName)]
```

Comment:
@kichalla do we want to add a test for `DotNetCore31SdkVersion` as well? If not, is there any reason we're just testing the preview version here?

Reply:
The reason is we depend on the output from the `dotnet --version` command which prints out the preview name as it is still in preview